app/services/idv/session.rb

```diff
@@ -65,7 +65,7 @@ def vendor_params
     end
 
     def profile
-      @_profile ||= Profile.find(profile_id)
+      @_profile ||= Profile.find_by(id: profile_id)
```

Comment:
This fixes a bug that was found when re-organizing the confirmation controller specs. It looks like we expect this to return nil if there is no profile. That implies that `profile_id` is nil. `Profile.find(nil)` raises an error which the user sees as a 500.